src/hosted-form/hosted-form-order-data-transformer.ts

```diff
@@ -42,4 +43,15 @@ export default class HostedFormOrderDataTransformer {
             paymentMethodMeta,
         };
     }
+
+    private _getAdditionalAction(paymentRecaptchaToken?: string) {
+        return paymentRecaptchaToken ?
+            {
+                type: 'recaptcha_v2_verification',
```

Comment:
If the handler class / function returns a payload object in this format instead of a string token, you probably won't need to have the same payload formatting logic in two different places.